src/components/label/Label.jsx

```diff
@@ -1,13 +1,16 @@
 import { useState } from "react";
-import { useNotes } from "../../context/context";
+import { useNotes, useDisplay } from "../../context/context";
 import "./Label.css";
 
-export const Label = ({ viewPopup }) => {
+export const Label = () => {
+  const {
+    displayState: { labelToggle },
+  } = useDisplay();
```

Comment:
just a suggestion - since there are different toggles in the code for labels colors etc, you may try to create a custom hook for toggle and use that wherever you want to toggle.

Reply:
will try it out.!